src/EventSubscriber/Actions/InvoiceSubscriber.php

```diff
@@ -31,10 +31,10 @@ public function onActions(PageActionsEvent $event): void
         }
 
         if ($this->isGranted('history_invoice')) {
-            if ($invoice->isNew()) {
+            if ($invoice->isNew() || $invoice->isPaid()) {
```

Comment:
What is your thought on this change?

Reply:
In testing, I created a lot of invoices to go through the states. It would be much easier if you could set invoices in the status `paid` back to `pending.` 

Then I thought this is a valid use case also for prod environments. For example, if you accidentally chose a wrong payment date, or set the wrong invoice to `paid` and you want to correct that mistake.